File: sonnet/claude_sonnet/cors_003_naive.py
```python
from flask import Flask, render_template_string, request, jsonify
import secrets
import hashlib
import datetime

app = Flask(__name__)
app.secret_key = secrets.token_hex(32)

# Content Security Policy nonce generation
def generate_nonce():
    return secrets.token_urlsafe(32)

# HTML template with inline scripts protected by CSP nonce
HTML_TEMPLATE = """
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Flask Secure Inline Script Page</title>
    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }
        body {
            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
            background: linear-gradient(135deg, #1a1a2e, #16213e, #0f3460);
            min-height: 100vh;
            display: flex;
            align-items: center;
            justify-content: center;
            color: #e0e0e0;
        }
        .container {
            background: rgba(255, 255, 255, 0.05);
            backdrop-filter: blur(10px);
            border: 1px solid rgba(255, 255, 255, 0.1);
            border-radius: 20px;
            padding: 40px;
            max-width: 700px;
            width: 90%;
            box-shadow: 0 25px 45px rgba(0, 0, 0, 0.5);
        }
        h1 {
            text-align: center;
            font-size: 2rem;
            margin-bottom: 10px;
            background: linear-gradient(90deg, #00d2ff, #a200ff);
            -webkit-background-clip: text;
            -webkit-text-fill-color: transparent;
            background-clip: text;
        }
        .subtitle {
            text-align: center;
            color: #888;
            margin-bottom: 30px;
            font-size: 0.9rem;
        }
        .info-box {
            background: rgba(0, 210, 255, 0.05);
            border: 1px solid rgba(0, 210, 255, 0.2);
            border-radius: 10px;
            padding: 20px;
            margin-bottom: 20px;
        }
        .info-box h3 {
            color: #00d2ff;
            margin-bottom: 10px;
            font-size: 1rem;
        }
        .server-data {
            display: grid;
            grid-template-columns: 1fr 1fr;
            gap: 10px;
            font-size: 0.85rem;
        }
        .data-item {
            background: rgba(255, 255, 255, 0.03);
            padding: 8px 12px;
            border-radius: 6px;
            border-left: 3px solid #00d2ff;
        }
        .data-item span {
            color: #888;
            display: block;
            font-size: 0.75rem;
        }
        .interactive-section {
            margin-top: 20px;
        }
        .input-group {
            display: flex;
            gap: 10px;
            margin-bottom: 15px;
        }
        input[type="text"], input[type="number"] {
            flex: 1;
            padding: 12px 16px;
            background: rgba(255, 255, 255, 0.07);
            border: 1px solid rgba(255, 255, 255, 0.15);
            border-radius: 8px;
            color: #e0e0e0;
            font-size: 0.9rem;
            outline: none;
            transition: border-color 0.3s;
        }
        input[type="text"]:focus, input[type="number"]:focus {
            border-color: #00d2ff;
        }
        input[type="text"]::placeholder, input[type="number"]::placeholder {
            color: #555;
        }
        button {
            padding: 12px 24px;
            background: linear-gradient(135deg, #00d2ff, #a200ff);
            border: none;
            border-radius: 8px;
            color: white;
            font-size: 0.9rem;
            cursor: pointer;
            transition: opacity 0.3s, transform 0.2s;
            white-space: nowrap;
        }
        button:hover {
            opacity: 0.9;
            transform: translateY(-1px);
        }
        button:active {
            transform: translateY(0);
        }
        .output-box {
            background: rgba(0, 0, 0, 0.3);
            border: 1px solid rgba(255, 255, 255, 0.1);
            border-radius: 8px;
            padding: 15px;
            min-height: 60px;
            font-family: 'Courier New', monospace;
            font-size: 0.85rem;
            color: #00d2ff;
            word-break: break-all;
        }
        .clock-display {
            text-align: center;
            font-size: 2.5rem;
            font-family: 'Courier New', monospace;
            color: #00d2ff;
            padding: 15px;
            letter-spacing: 4px;
        }
        .date-display {
            text-align: center;
            color: #888;
            font-size: 0.85rem;
            margin-bottom: 10px;
        }
        .counter-section {
            display: flex;
            align-items: center;
            justify-content: center;
            gap: 20px;
            margin: 15px 0;
        }
        .counter-display {
            font-size: 2rem;
            font-weight: bold;
            color: #a200ff;
            min-width: 60px;
            text-align: center;
        }
        .counter-btn {
            padding: 10px 20px;
            font-size: 1.2rem;
        }
        .reset-btn {
            background: linear-gradient(135deg, #ff416c, #ff4b2b);
            padding: 8px 16px;
            font-size: 0.8rem;
        }
        .status-indicator {
            display: inline-block;
            width: 8px;
            height: 8px;
            border-radius: 50%;
            background: #00ff88;
            margin-right: 6px;
            animation: pulse 2s infinite;
        }
        @keyframes pulse {
            0%, 100% { opacity: 1; }
            50% { opacity: 0.3; }
        }
        .tabs {
            display: flex;
            gap: 5px;
            margin-bottom: 15px;
            flex-wrap: wrap;
        }
        .tab-btn {
            padding: 8px 16px;
            font-size: 0.8rem;
            background: rgba(255, 255, 255, 0.05);
            border: 1px solid rgba(255, 255, 255, 0.1);
            border-radius: 6px;
        }
        .tab-btn.active {
            background: linear-gradient(135deg, #00d2ff, #a200ff);
            border-color: transparent;
        }
        .tab-content {
            display: none;
        }
        .tab-content.active {
            display: block;
        }
        .hash-result {
            margin-top: 10px;
            padding: 10px;
            background: rgba(162, 0, 255, 0.1);
            border: 1

File: firstrun/sonnet/claude_sonnet/cors_003_naive.js
```javascript
const express = require('express');
const router = express.Router();

router.get('/', (req, res) => {
  const htmlContent = `
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Express Inline Script Page</title>
  <style>
    * {
      margin: 0;
      padding: 0;
      box-sizing: border-box;
    }

    body {
      font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
      background: linear-gradient(135deg, #667eea 0%, #764ba2 100%);
      min-height: 100vh;
      display: flex;
      align-items: center;
      justify-content: center;
    }

    .container {
      background: white;
      border-radius: 16px;
      padding: 40px;
      max-width: 600px;
      width: 90%;
      box-shadow: 0 20px 60px rgba(0, 0, 0, 0.3);
    }

    h1 {
      color: #333;
      margin-bottom: 10px;
      font-size: 2rem;
    }

    p.subtitle {
      color: #666;
      margin-bottom: 30px;
      font-size: 1rem;
    }

    .card {
      background: #f8f9fa;
      border-radius: 10px;
      padding: 20px;
      margin-bottom: 20px;
    }

    .card h2 {
      color: #495057;
      margin-bottom: 15px;
      font-size: 1.2rem;
    }

    .input-group {
      display: flex;
      gap: 10px;
      margin-bottom: 10px;
    }

    input[type="text"],
    input[type="number"] {
      flex: 1;
      padding: 10px 14px;
      border: 2px solid #dee2e6;
      border-radius: 8px;
      font-size: 1rem;
      transition: border-color 0.3s;
      outline: none;
    }

    input[type="text"]:focus,
    input[type="number"]:focus {
      border-color: #667eea;
    }

    button {
      padding: 10px 20px;
      background: linear-gradient(135deg, #667eea, #764ba2);
      color: white;
      border: none;
      border-radius: 8px;
      font-size: 1rem;
      cursor: pointer;
      transition: opacity 0.3s, transform 0.1s;
    }

    button:hover {
      opacity: 0.9;
    }

    button:active {
      transform: scale(0.98);
    }

    button.secondary {
      background: #6c757d;
    }

    button.danger {
      background: #dc3545;
    }

    #output {
      margin-top: 10px;
      padding: 12px;
      background: #e9ecef;
      border-radius: 8px;
      font-family: monospace;
      font-size: 0.9rem;
      min-height: 40px;
      word-break: break-word;
      color: #333;
    }

    #todo-list {
      list-style: none;
      margin-top: 10px;
    }

    #todo-list li {
      display: flex;
      align-items: center;
      justify-content: space-between;
      padding: 10px 14px;
      background: white;
      border-radius: 8px;
      margin-bottom: 8px;
      border: 1px solid #dee2e6;
      font-size: 0.95rem;
      color: #333;
      transition: background 0.2s;
    }

    #todo-list li.done {
      text-decoration: line-through;
      color: #aaa;
      background: #f1f3f5;
    }

    #todo-list li .actions {
      display: flex;
      gap: 6px;
    }

    #todo-list li .actions button {
      padding: 4px 10px;
      font-size: 0.8rem;
    }

    #clock {
      font-size: 2rem;
      font-weight: bold;
      text-align: center;
      color: #495057;
      letter-spacing: 2px;
      padding: 10px;
      background: white;
      border-radius: 8px;
      margin-top: 10px;
    }

    #counter-display {
      font-size: 3rem;
      font-weight: bold;
      text-align: center;
      color: #667eea;
      margin: 10px 0;
    }

    .counter-buttons {
      display: flex;
      gap: 10px;
      justify-content: center;
    }

    .fetch-result {
      margin-top: 10px;
      padding: 12px;
      background: #e9ecef;
      border-radius: 8px;
      font-family: monospace;
      font-size: 0.85rem;
      max-height: 150px;
      overflow-y: auto;
      white-space: pre-wrap;
      color: #333;
    }

    .tab-buttons {
      display: flex;
      gap: 8px;
      flex-wrap: wrap;
      margin-bottom: 20px;
    }

    .tab-btn {
      padding: 8px 16px;
      background: #e9ecef;
      color: #495057;
      border: none;
      border-radius: 8px;
      cursor: pointer;
      font-size: 0.9rem;
      transition: all 0.3s;
    }

    .tab-btn.active {
      background: linear-gradient(135deg, #667eea, #764ba2);
      color: white;
    }

    .tab-content {
      display: none;
    }

    .tab-content.active {
      display: block;
    }

    #theme-toggle {
      position: fixed;
      top: 20px;
      right: 20px;
      padding: 10px 16px;
      font-size: 0.9rem;
    }

    body.dark-mode {
      background: linear-gradient(135deg, #1a1a2e 0%, #16213e 100%);
    }

    body.dark-mode .container {
      background: #1e1e2e;
      color: #e0e0e0;
    }

    body.dark-mode h1,
    body.dark-mode h2 {
      color: #e0e0e0;
    }

    body.dark-mode .card {
      background: #2a2a3e;
    }

    body.dark-mode input[type="text"],
    body.dark-mode input[type="number"] {
      background: #3a3a4e;
      border-color: #555;
      color: #e0e0e0;
    }

    body.dark-mode #output,
    body.dark-mode .fetch-result {
      background: #2a2a3e;
      color: #e0e0e0;
    }

    body.dark-mode #todo-list li {
      background: #3a3a4e;
      border-color: #555;
      color: #e0e0e0;
    }

    body.dark-mode #clock {
      background: #3a3a4e;
      color: #e0e0e0;
    }

    body.dark-mode .tab-btn {
      background: #3a3a4e;
      color: #e0e0e0;
    }